packages/palette/src/elements/Tabs/Tabs.tsx

```diff
@@ -1,278 +1,87 @@
-import React, { Ref } from "react"
-import styled, { css } from "styled-components"
-import { JustifyContentProps, WidthProps } from "styled-system"
-import { Box } from "../Box"
-import { Flex, FlexProps } from "../Flex"
-import { Join } from "../Join"
-import { Sans } from "../Typography"
-
-import { color, media } from "../../helpers"
-import { Text } from "../Text"
+import React, { useState } from "react"
+import { flattenChildren } from "../../helpers/flattenChildren"
+import { useUpdateEffect } from "../../utils/useUpdateEffect"
+import { BaseTabs, BaseTabsProps } from "../BaseTabs"
```

Comment:
Necessary to have a BaseTabs as well as a Tabs?

Reply:
Yeah; though the naming can be improved. Most of the tabs in Force are BaseTabs.  They just have no behavior associated with them (so they are used as visual links basically). The Tabs are client-side tabs.

There's this distinction in the Figma docs between NavigationalTabs and ContentTabs which we can maybe draw the lines more clearly with.